implementations/rust/ockam/ockam_command/src/run/runner.rs

```diff
@@ -0,0 +1,51 @@
+use crate::run::parser::{ArgsToCommands, Config};
+use crate::CommandGlobalOpts;
+
+use miette::Result;
+use ockam_node::Context;
+
+pub struct ConfigRunner;
+
+impl ConfigRunner {
```

Comment:
A general comment explaining the overall strategy for running commands described in files would be great here:

 - we're parsing some yaml content, where the keys are expected to be resource + command names (with an example of how `ockam node create` would appear in the file)
 - we're parsing that yaml content as a command as if it was read from the command line
 - we're executing all the commands in the file in the proper order (nodes are created before inlets/outlets for example)

Reply:
I've added some details, but I'll extend them in the next PR, as I'm adding some changes that will need an update of the docs.